ksql-streams/src/test/java/io/confluent/ksql/execution/streams/StreamSinkBuilderTest.java

```diff
@@ -90,12 +92,14 @@ public void setup() {
     when(keySerdeFactory.buildKeySerde(any(), any(), any())).thenReturn(keySerde);
     when(queryBuilder.buildValueSerde(any(), any(), any())).thenReturn(valSerde);
     when(source.build(any())).thenReturn(new KStreamHolder<>(kStream, SCHEMA, keySerdeFactory));
+    doReturn(kStream).when(kStream).transform(any());
```

Comment:
This test class and `TableSinkBuilderTest` are now both essentially just testing the new `SinkBuilder` class.  

Move the tests from these two test classes into `SinkBuilderTest` and remove them.

Reply:
Done